# reference path
# track_points_x: [0,0.5, 1,1.7, 2, 1.7,  1,0, -1,-1.7,-2,-2.3,-3.0, -3.7, -4.0, -4.0, -4.0]
# track_points_y: [1,  1, 1,0.7, 0,-0.7,-1,-1, -1,-0.7, 0, 0.7, 1.0,  0.7,  0.0, -0.5, -1.0]
# track_points_x: [0.0 ,0.25, 0.5 , 0.85, 1, 0.85,  0.5,   0, -0.5, -0.85,-1,-1.15,-1.5, -1.85, -2.0, -2.0, -2.0]
# track_points_y: [0.5 , 0.5, 0.5 , 0.35, 0,-0.35, -0.5,-0.5, -0.5, -0.35, 0, 0.35, 0.5,  0.35,  0.0, -0.25, -0.5]

desired_v: 1 # desired velocity

dt: 0.005
rho: 0 # J = (x-x_r)^2+ (y-y_r)^2 + rho * (phi-phi_r)^2
N: 50 #horizon
rhoN: 1

# lower and upper bound
v_max: 1
a_max: 4
w_max: 1
alp_max: 1
j_max: 40
# ctrl delay
delay: 0.00001

kl: 1.0
km: 1
ks: 1
